src/Symfony/Component/Validator/Validation.php

```diff
@@ -49,6 +49,26 @@ public static function createCallable($constraintOrValidator = null, Constraint
         };
     }
 
+    /**
+     * Creates a callable that returns true/false instead of throwing validation exceptions.
+     *
+     * @param Constraint|ValidatorInterface|null $constraintOrValidator
+     */
+    public static function createValidCallable($constraintOrValidator = null, Constraint ...$constraints): callable
```

Comment:
`createIsValidCallable`?

Reply:
Maybe `createBooleanCallable()`? I'm surprised there is no "official" term for this type of function, validation using return vs exception seems quite a common pattern/decision.

Reply:
we obtain a validator function: Validation::createValidatorCallable
we obtain a validation function: Validation::createCallable